docs/modules/serialization/pages/serialization.adoc

```diff
@@ -68,6 +68,25 @@ may incur costs on short-lived use cases
 |===
 | Serialization interface| Advantages| Disadvantages|Client support
 
+| xref:implementing-dataserializable.adoc[`IdentifiedDataSerializable`]
+| * More CPU and memory usage efficient than `Serializable`
+
+* Reflection is not used during deserialization
+
+| * Specific to Hazelcast
+
+* Serialization interface must be implemented
+
+* A factory and configuration must be implemented
```

Comment:
What is a factory? I see "A Factory and configuration must be implemented" used elsewhere on page